app/api-fetch/all-albums.ts

```
@@ -3,35 +3,64 @@
 export default async function fetchAllAlbums() {
     if (!process.env.NEXT_PUBLIC_API_URL) {
         throw new Error('API URL not set');
     }
     try {
-        if (localStorage.getItem("albums") !== null || Date.now() - parseInt(localStorage.getItem("albumsTime")!) < 3600000) {
-            const data = AlbumSchema.parse(JSON.parse(localStorage.getItem("albums")!));
-            return data["data"] as Album[];
+        const storedAlbums = localStorage.getItem("albums");
+        const storedTime = localStorage.getItem("albumsTime");
+        
+        if (storedAlbums && storedTime && Date.now() - parseInt(storedTime) < 3600000) {
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)